fighthealthinsurance/views.py

```
@@ -1998,39 +1998,78 @@ def do_post(self, request):
+            wants_json = request.GET.get("format") == "json"
+            if error is not None or next_url is None:
+                message, status_code = error or ("An internal error occurred", 500)
+                if wants_json:
+                    return self._json_error_response(message, status_code)
+                return self._render_error_page(request, message, status_code)
+            if wants_json:
+                return JsonResponse({"next_url": next_url})
+            # Browser navigation from the recovery email: send the visitor
+            # straight to Stripe rather than showing them raw JSON.
+            return HttpResponseRedirect(next_url)
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)